scipy/optimize/optimize.py

```diff
@@ -111,10 +111,10 @@ class OptimizeResult(dict):
 
     Notes
     -----
-    There may be additional attributes not listed above depending of the
-    specific solver. Since this class is essentially a subclass of dict
-    with attribute accessors, one can see which attributes are available
-    using the `keys()` method.
+    `OptimizeResult` may have additional attributes not listed here depending
+    on the specific solver being used. Since this class is essentially a
+    subclass of dict with attribute accessors, one can see which
+    attributes are available using the `OptimizeResult.keys` method.
```

Comment:
```suggestion
    attributes are available using the :func:`OptimizeResult.keys` method.
```

Not sure if we also need a `.. currentmodule::` at the top. If it is created via `.. autosummary::` (I think so), then no, otherwise we have to add it. I will check how the docs render...

Reply:
I think `:func:` is unnecessary here.